cclib/parser/data.py

```diff
@@ -135,32 +138,53 @@ class ccData:
        "fooverlaps":       Attribute(numpy.ndarray,    'orbital overlap',             'fragments'),
        "fragnames":        Attribute(list,             'fragment names',              'fragments'),
        "frags":            Attribute(list,             'atom indices',                'fragments'),
+       "functional":       Attribute(str,              'functional',                  'metadata'),
+       "g4_energy":        Attribute(float,            'G4 energy',                   'properties:energy'),

```

Comment:
right, the Gn series are composite methods that include B3LYPwith some post-HF methods. I can't help you much here, but (if it helps) the final energy isn't any of the values of the individual methods but a final value obtained with corrections using the individual calculations. The Gn energy is printed separately and it needs to be adjusted since it includes ZPE. For example, in G4 the energy + ZPE is shown as:

`G4(0 K)=            -76.332054`

Reply:
Ok thanks for the explanation, is the ZPE intrinsically part of the G4 energy or does it also have meaning without ZPE? Just trying to think whether the G4 energy we parse should include ZPE or not, as ZPE should be available separately

Reply:
good point - in the original version, we separated E from ZPE. But after some discussions, we thought that it might be better to leave E+ZPE as the Gn methods might have been designed to include ZPE. 

Long story short, we're still not sure of what's the best way to parse these energies (either E or E+ZPE).

Reply:
In that case then, best to have both :)